src/mixins/DrawingTool.js

```diff
@@ -499,16 +499,17 @@ const ThreePointsDrawingTool = DrawingTool.named('ThreePointsDrawingTool')
         const shape = self.getCurrentArea();
 
         if (!shape) return;
-        const { stageWidth, stageHeight } = self.obj;
+        const maxStageWidth = isFF(FF_DEV_3793) ? 100 : self.obj.stageWidth;
+        const maxStageHeight = isFF(FF_DEV_3793) ? 100 : self.obj.stageHeight;
```

Comment:
can you put the 100 in a variable and a comment to help explain why 100

Reply:
I think that 100 is used in src/regions/RectRegion.js aswell - would it make sense to define it as a constant value so we can reuse it where needed - would help the code be more readable